opensrp-stock/src/main/java/org/smartregister/stock/helper/SyncStockTypeServiceHelper.java

```diff
@@ -69,32 +72,63 @@ public void pullStockTypeFromServer() {
                     Timber.e(e);
                 }
             } else {
-                sendSyncStatusBroadcastMessage(FetchStatus.fetchStarted);
-                String baseUrl = getFormattedBaseUrl();
-                Response<String> response = httpAgent.fetch(String.format("%s%s", baseUrl, SYNC_URL));
-                if (response != null && response.payload() != null) {
-                    saveAllStockTypes(response.payload());
-                    if (stockSyncConfiguration.shouldFetchStockTypeImages()) {
-                        downloadStockTypeImages();
+                SharedPreferences preferences = PreferenceManager.getDefaultSharedPreferences(context);
```

Comment:
Consider accessing SharedPreferences through the CoreLibrary instance.
Note: Option to encrypt SharedPreferences has been added in Core [not merged yet]. Apps are able to control whether to to encrypt or not.

Reply:
noted and updated